lib/select.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'menu.rb'
+
+class Select
+
+	attr_reader :order, :menu
+
+	def initialize
+		@menu = Menu.new
+		@menu = @menu.menu
+		@order = {}
+	end
+	def see
+		@menu
+	end
+    
+    def pick(dish , quantity=1)
+    	#@menu[dish]
+    	raise 'this dish is not in the menu' unless @menu.include? dish
+    	@order.merge!({dish => quantity})
+    end
+    def bill
+    	@order.inject(0) {|key,value| puts value}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)